core/blockly_options.ts

```diff
@@ -15,11 +15,68 @@
 import * as goog from '../closure/goog/goog.js';
 goog.declareModuleId('Blockly.BlocklyOptions');
 
+import {Theme} from './theme.js';
+import {Workspace} from './workspace.js';
+
 
 /**
  * Blockly options.
  * This interface is further described in
  * `typings/parts/blockly-interfaces.d.ts`.
```

Comment:
Remove outdated comment, please

Just for reference. BlocklyOptions used to be manually typed with that referenced file, before we switched over to goog.module. Part of the typings script added the type information from that file to the overall output of the tool we were using to generate the .d.ts files. That functionality wasn't carried over when we switched to using tsc (until this change, yay!)